core/src/main/scala/kafka/tools/GetOffsetShell.scala

```diff
@@ -70,12 +71,13 @@ object GetOffsetShell {
                            .withRequiredArg
                            .describedAs("partition ids")
                            .ofType(classOf[String])
-    val timeOpt = parser.accepts("time", "timestamp of the offsets before that. [Note: No offset is returned, if the timestamp greater than recently committed record timestamp is given.]")
+    val timeOpt = parser.accepts("time", "timestamp of the offsets before that. [Note: No offset is returned, if the timestamp greater than recently committed record timestamp is given.]" +
+                                  " It also accepts \"earliest\", \"latest\", \"max-timestamp\", -1(latest), -2(earliest) -3(max timestamp) ")
                            .withRequiredArg
-                           .describedAs("timestamp/-1(latest)/-2(earliest)")
-                           .ofType(classOf[java.lang.Long])
-                           .defaultsTo(-1L)
-    val commandConfigOpt = parser.accepts("command-config", s"Property file containing configs to be passed to Consumer Client.")
+                           .describedAs("timestamp/latest/earliest/max-timestamp/-1(latest)/-2(earliest)/-3(max timestamp)")
```

Comment:
nit: I wonder if we could just say: `<timestamp> / -1 or latest / -2 or earliest / -3 or max-timestamp`. What do you think?